utils/utils.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+	"errors"
+	"math"
+	"os/exec"
+	"runtime"
+	_time "time"
+)
+
+type Utils interface {
+	Now() _time.Time
+	OpenBrowser(url string) error
+}
+
+type utils struct{}
+
+func NewUtils() Utils {
+	return &utils{}
+}
+
+func (u *utils) Now() _time.Time {
+	return _time.Now()
+}
+
+func (u *utils) OpenBrowser(url string) error {
+	switch runtime.GOOS {
+	case "linux":
+		return exec.Command("xdg-open", url).Start()
```

Comment:
Can we additionally add WSL detection here? It's reported as Linux, but the command to open should like on Windows (but needs the `exe` extension - `rundll32.exe`).

Some recommendations here but open to other ways: https://github.com/microsoft/WSL/issues/423#issuecomment-221627364